commerce-frontend-js/src/main/resources/META-INF/resources/components/modal/Modal.es.js

```diff
@@ -65,13 +71,13 @@ const Modal = props => {
 	}
 
 	useEffect(() => {
-		switch (true) {
-			case iframeLoadingCounter > 1:
-				return _handleFormSubmit();
-			case iframeLoadingCounter === 1:
-				return _handleIframeFirstLoad();
-			default:
-				break;
+		
+		if(iframeLoadingCounter > 1) {
+			_handleFormSubmit();
+		}
+		
+		if(iframeLoadingCounter === 1) {
+			_handleIframeFirstLoad();
 		}
 		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
do we need this comment?

Reply:
Not mandatory but it hides a eslint warning. I would keep it there